client/scripts/ui.js

```diff
@@ -266,10 +267,23 @@ class ReceiveDialog extends Dialog {
             $a.click()
             return
         }
-        if(file.mime.split('/')[0] === 'image'){
-            console.log('the file is image');
-            this.$el.querySelector('.preview').style.visibility = 'inherit';
-            this.$el.querySelector("#img-preview").src = url;
+        
+        let mine = file.mime.split('/')[0]
```

Comment:
Assigned a wrong name to this var, it should be "mime", not "mine"... (that's purely for better code comprehension)

Reply:
THANKS.. Sorry for my english 😂